installers/custom_ipxe/ipxe_script_test.go

```diff
@@ -27,7 +27,7 @@ func TestScript(t *testing.T) {
 		s := ipxe.Script{}
 		s.Echo("Packet.net Baremetal - iPXE boot")
 		s.Set("iface", "eth0").Or("shell")
-		s.Set("tinkerbell", "http://127.0.0.1")
+		s.Set("boots", "http://127.0.0.1")
```

Comment:
this needs to stay tinkerbell to remain backwards compatible with any ipxe scripts out there in the wild